cmd/zetae2e/local/utils.go

```diff
@@ -2,10 +2,13 @@ package local
 
 import (
 	"context"
+	"errors"
```

Comment:
I think we can create a separate file `monitor_priority_txs.go`

We should remove `utils.go` btw